### ISSUE TITLE
Error: HTTP status client error (429 Too Many Requests)

### ISSUE THREAD

**Issue by fossdd** [rylev/triage-tracker#3]
I've ran:
```
cargo run closings range -s 2021-06-07 -e 2021-05-31
```
like described in the README.md but I got raid-limited. Do I have to pay to GitHub or sign-in to get a longer range?

**Comment by rylev** [comment]
Yes, GitHub only offers a limited number of requests. I'd like to offer the ability to sign into your account in order to get more requests (though this is still rate limited unfortunately).
